### ISSUE TITLE
adb_stop_bit_srq_listen() and adb_wait_tlt()

### ISSUE THREAD

**Issue by arpruss** [szymonlopaciuk/stm32-adb2usb#1]
Looking at the code, the two functions always return the same value.

**Comment by szymonlopaciuk** [comment]
You are right, also about #2. Sorry for the delay, I am a bit busy at the moment. Feel free to make a pull request as you see fit, or otherwise I will try to fix in at some point when I have more time.

**Comment by arpruss** [comment]
It's too much work to make the pull request right now, as my own fork revises the code very heavily to make it work with the less bloated libmaple core (using my USBComposite library) and to use the CYCCNT debug register for accurate timing instead of calling delayMicroseconds(). At least on libmaple, and maybe on the official stm32 core, delayMicroseconds() is wrong on some stm32f103c8t6 clones. Also, I have found CYCCNT to be an excellent way to handle timing, as you can avoid worries about overhead. My fork is: https://github.com/arpruss/stm32-adb2usb/
